deploy/kube-config/riemann/heapster-controller.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Can we avoid adding a separate controller? I'm trying to get rid of the existing ones and document using the remote API for enabling sinks. For now, just documenting the reimann sink should work I think.